client/dashboard/components/date-range-picker/date-range-content.tsx

```diff
@@ -0,0 +1,218 @@
+import { DateRangeCalendar, TZDate } from '@automattic/ui';
+import {
+	__experimentalText as Text,
+	Button,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	VisuallyHidden,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { formatYmd } from '../../utils/datetime';
+import { DateInputs } from './date-inputs';
+import { PresetsListbox } from './presets-listbox';
+import { computePresetRange, getActivePresetId, PresetId } from './utils';
+
+type DateRangeContentProps = {
+	isSmall: boolean;
+	fromDraft?: Date;
+	toDraft?: Date;
+	fromStr: string;
+	toStr: string;
+	setFromDraft: ( date?: Date ) => void;
+	setToDraft: ( date?: Date ) => void;
+	setFromStr: ( string: string ) => void;
+	setToStr: ( string: string ) => void;
+	timezoneString?: string;
+	gmtOffset?: number;
+	onChange: ( next: { start: Date; end: Date } ) => void;
+	onClose?: () => void;
+	compositeActiveId: string | null;
+	setCompositeActiveId: ( id: string | null ) => void;
+	siteToday: Date;
+	siteTodayStr: string;
```

Comment:
It seems weird that there is `site` concept here. Would it be better to call it `today`?

Reply:
The idea is that this should be a `DateRangePicker` rather than a `SiteDateRangePicker`, so we shouldn’t include anything related to “site.” The same applies to the siteToday variable in `DateRangePickerInner`. I think both `timezoneString` and `gmtOffset` can be used to determine today based on the site. Does it make sense?